cmd/tgrade/root.go

```diff
@@ -123,6 +122,19 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig appparams.EncodingConfig
 	)
 }
 
+// customize init config
+func customizeInitCmdDefaults(initCmd *cobra.Command) *cobra.Command {
+	orig := initCmd.RunE
+	initCmd.RunE = func(cmd *cobra.Command, args []string) error {
+		cfg := server.GetServerContextFromCmd(cmd)
+		// set a high default value to have a more resilient network but this also puts a higher strain on node memory
+		cfg.Config.P2P.MaxNumOutboundPeers = 200
+		server.SetCmdServerContext(cmd, cfg)
+		return orig(initCmd, args)
```

Comment:
This basically installs this custom hook over the original cmd.

Reply:
yes, this works for now but not very readable. They prepared an extension point in the current sdk master already. 🎉